modopt/opt/algorithms.py

```diff
@@ -152,6 +152,8 @@ def metrics(self, metrics):
             raise TypeError(
                 'Metrics must be a dictionary, not {0}.'.format(type(metrics)),
             )
+        else:
+            self._metrics = metrics
```

Comment:
This is obviously fine in itself. The reason the CI tests fail is because [WPS doesn't like negated conditions](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/refactoring.html#wemake_python_styleguide.violations.refactoring.NegatedConditionsViolation). To resolve simply swap the `elif` and `else` statements without the `not`.

Reply:
Yes @sfarrens , I have done just this in my next PR. We can ideally close this if the other one is merged